tests/db/model/token/delete.js

```diff
@@ -40,6 +40,28 @@ describe('db: Token: find', () => {
       expect(rToken).to.be.equal(null);
       done();
     })
-    .catch((err) => done(err));
+    .catch(done);
+  });
+});
+
+describe('db: System Token: delete', () => {
+  let t;
+  beforeEach((done) => {
+    u.createSystemToken()
+    .then((token) => {
+      t = token;
+      done();
+    })
+    .catch(done);
+  });
+  afterEach(u.forceDelete);
+  it('Not allowed to delete system token', (done) => {
+    Token.findOne({ where: { name: t.name } })
+    .then((token) => token.destroy())
+    .then(() => done('expecting error here'))
```

Comment:
We don't want to let a user delete the system-generated token for their user record, so the token.destroy call should send back an error